src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -123,7 +127,7 @@ static ConnectionProvider elastic(String name, @Nullable Duration maxIdleTime, @
 	 * number of {@link Connection}
 	 */
 	static ConnectionProvider fixed(String name) {
-		return fixed(name, DEFAULT_POOL_MAX_CONNECTIONS);
```

Comment:
Keep this and invoke the builder creation just once in `fixed(String name, int maxConnections, long acquireTimeout, @Nullable Duration maxIdleTime, @Nullable Duration maxLifeTime)`